lib/travis/logs/services/process_log_part.rb

```diff
@@ -65,14 +66,13 @@ def run(payload)
         end
 
         private def valid_log_id?(log_id, payload)
-          if log_id == 0
-            Travis.logger.warn(
-              'invalid log id',
-              action: 'process', job_id: payload['id'],
-              result: 'invalid_id', log_id: log_id
-            )
-            mark('log.id_invalid')
-          end
+          return true unless log_id.zero?
```

Comment:
Oh good heavens, I was mixing this up with some other code.  Indeed, it looks like the return value is not used here.  Changing...